preface.asciidoc

```diff
@@ -184,4 +184,5 @@ Following is a list of notable GitHub contributors, including their GitHub ID in
   - Alex Frolov (sashafrolov)
   - Richard Sun (richardsfc)
   - Brian Leffew (bleffew99)
-* ztz (zt2)
+* Solomon Victorino (bitsol)
+* ztz (zt2)
```

Comment:
My first commit: 070d848db5aeace754491e9db4d94139d878e9d9, had nothing but CornellBlockchain and its members at the end, so I believe that justifies my name being here.
[Proof of working tree.](https://github.com/bitsol/ethereumbook/blob/070d848db5aeace754491e9db4d94139d878e9d9/preface.asciidoc) ;)